### PR TITLE
[FEATURE] Add "ledger" field to Account

### PR DESCRIPTION
**Pull request by sentientwaffle**
For https://github.com/interledger/five-bells-connector/issues/51 and https://github.com/interledger/five-bells-connector/issues/50 and https://github.com/interledger/five-bells-ledger/issues/105

### CODE REVIEW

File: schemas/Account.json [hunk 1]
```diff
@@ -35,6 +35,10 @@
     "is_disabled": {
       "description": "Admin users may disable/enable an account",
       "type": "boolean"
+    },
+    "ledger": {
+      "description": "A link the the account's ledger",
+      "type": "Iri.json"
     }
   },
   "required": ["name", "balance"],
```
Comment (clark800) [inline schemas/Account.json]: Did you want to remove "balance"?

Comment (sentientwaffle) [inline schemas/Account.json]: Good point; unauthorized users will get the account without the balance property.

**Comment by MatthewPhinney** [comment]
I may be missing something, but on the ledger side, isn't this schema only used to validate incoming `PUT` requests? I thought this PR was about allowing unauthorized users to `GET` basic info about an account, in which case you can return any object you want (it doesn't look like the response is validated by this schema). Is this schema used elsewhere?

**Comment by sentientwaffle** [comment]
You're correct, but I think from a REST point of view we want the objects returned from `/foo_objects/:id` to match the `FooObject` schema.

**Comment by clark800** [comment]
This is something we should test, I added an issue: https://github.com/interledger/five-bells-ledger/issues/107

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**